lib/rambling/trie/version.rb

```diff
@@ -3,6 +3,6 @@
 module Rambling
   module Trie
     # Current version of the rambling-trie.
-    VERSION = '2.1.2'
+    VERSION = '2.1.1'
```

Comment:
What happened here?

Reply:
I've worked from version 2.1.1 and the latest one 2.1.2

Reply:
Are we doing 2.1.2 or 2.1.1? Why not 2.1.2?

Reply:
probably, I guess it's updated a few days ago, definitely after I started work.

Reply:
by the way, this is literally a new Gem created by us, so explicitly, the version number should start from again

Reply:
Let's use the same versioning since we plan to make a PR back to the original library.

Reply:
Can you apply the difference from rambling-trie 2.1.2 to this gem and update to 2.1.2? Thanks.

Reply:
sure